Sources/FluentUI_iOS/Components/Tab Bar/TabBarView.swift

```diff
@@ -92,11 +92,25 @@ open class TabBarView: UIView, TokenizedControl {
             stackView.setCustomSpacing(spacing, after: view)
         }
     }
+    /// Initializes MSTabBarView
+    /// - Parameter showsItemTitles: Determines whether or not to show the titles of the tab bar items.
+    @objc public convenience init(showsItemTitles: Bool = false) {
+        self.init(showsItemTitles: showsItemTitles, hideSeparator: false, disableBlur: false)
+    }
 
     /// Initializes MSTabBarView
     /// - Parameter showsItemTitles: Determines whether or not to show the titles of the tab bar items.
-    @objc public init(showsItemTitles: Bool = false) {
+    /// - Parameter hideSeparator: Removes the top divider displayed above the tab bar.
+    /// - Parameter disableBlur: Disables the blur effect applied to the tab bar.
+    @objc public init(showsItemTitles: Bool = false, hideSeparator: Bool = false, disableBlur: Bool = false) {
         self.showsItemTitles = showsItemTitles
+
+        if disableBlur {
+            self.backgroundView = UIVisualEffectView()
+        } else {
+          self.backgroundView = UIVisualEffectView(effect: UIBlurEffect(style: UIBlurEffect.Style.systemChromeMaterial))
+        }
```

Comment:
minor: I think this could be reduced to

```
var blurEffect: UIVisualEffect? = disableBlur ? nil : UIBlurEffect(style: UIBlurEffect.Style.systemChromeMaterial)
self.backgroundView = UIVisualEffectView(effect: blurEffect) 
```